src/app/render/render.go

```diff
@@ -94,5 +94,14 @@ func Render(cCtx *cli.Context) error {
 		return fmt.Errorf("rendering changelog: %w", err)
 	}
 
+	// At the time of writing this, this is a comment you can read in the changelog renderer:
+	// > For templates to be sane and readable, we need to put spaces between sections _after_ them. This comes with the
+	// > problem of the last section of the doc also printing those spaces, leading to two empty newlines.
+	// Because of that, rendered will output the markdown without a leading line jump that we have to add here.
+	_, err = fmt.Fprintf(mdFile, "\n")
```

Comment:
I would formulate this comment different because it was hard to follow for me, something like:

_As it's written in a comment in the renderer.Render function, we are putting spaces between sections \_after\_ them, this comes with the problem of the last section of the doc also printing those spaces, leading to two empty newlines, because of that renderer.Render will output the markdown without a leading line jump, so we add it here._